pkgs/development/python-modules/django-hcaptcha/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, django
+}:
+
+buildPythonPackage rec {
+  pname = "django-hcaptcha";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "AndrejZbin";
+    repo = "django-hCaptcha";
+    # FIXME: see https://github.com/AndrejZbin/django-hcaptcha/issues/11
+    rev = "29910669cffe783b2505f842c5da1842bbacd6aa";
+    sha256 = "sha256-g6I/CKxaRWuXLVhuAe1auC3+gHwcyYhoc/px+OfP4o8=";
+  };
```

Comment:
If the upstream doesn't have tests there is little reason to fetch from GitHub. In that case we then prefer `fetchPypi`, because it requires versioning.

Reply:
:+1: